src/Affects/Config/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "tenancy/affects-config",
+    "description": "The tenancy/tenancy config mutations",
+    "keywords": ["affects", "tenancy", "config"],
+    "license": "MIT",
+    "require": {
+        "tenancy/framework": "*"
+    },
+    "autoload": {
+        "psr-4": {
+            "Tenancy\\Affects\\Config\\": ""
+        }
+    },
+    "authors": [
+        {
+            "name": "Daniël Klabbers",
+            "email": "daniel@klabbers.email",
+            "homepage": "http://luceos.com"
+        }
```

Comment:
@ArlonAntonius someone is missing here
^

Reply:
Thanks :D